composer.json

```diff
@@ -13,7 +13,8 @@
     "require": {
         "php": ">=5.4",
         "php-http/httplug": "^1.0",
-        "php-http/message-factory": "^1.0"
+        "php-http/message-factory": "^1.0",
+        "symfony/options-resolver": "^2.6|^3.0"
```

Comment:
This is one of my concerns. Adding this dependency here. Maybe we should rethink option handling in the plugin client?